src/core/QuoridorPlayer.cs

```diff
@@ -15,7 +15,7 @@ internal class QuoridorPlayer
         private int y;
 
         public int AvailableWalls { get; }
```

Comment:
Use only one internal variable to represent amount of available walls that is initialized by the constructor